pysgrid/sgrid.py

```diff
@@ -599,51 +713,7 @@ def interpolation_alphas(self,
                              grid='center',
                              _memo=False,
                              _copy=False,
-                             _hash=None,):
-        """
-        Given an array of points, this function will return the bilinear
```

Comment:
Agreed, though it's going to take longer than I have time for now. In the interim I've added a top level summary of what the function does and the sources the math is based on, and some notes in the code for future reference